src/stubs/Test.stub

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace {{namespace}};
+
+use App\Http\Controllers\{{ucPluralModelName}}\{{modelName}}Controller;
```

Comment:
Please use ControllerNamespace instead of making up the namespace in the stub (check  RepositoriesNamespace in src/Commands/Generator.php:174 , it should look like it)